pkg/providers/snow/snow.go

```diff
@@ -72,7 +72,7 @@ func (p *SnowProvider) SetupAndValidateCreateCluster(ctx context.Context, cluste
 		return fmt.Errorf("setting defaults and validate snow config: %v", err)
 	}
 	if !p.skipIpCheck {
-		if err := providerValidator.ValidateControlPlaneIpUniqueness(clusterSpec.Cluster, &networkutils.DefaultNetClient{}); err != nil {
+		if err := validation.ValidateControlPlaneIpUniqueness(clusterSpec.Cluster, &networkutils.DefaultNetClient{}); err != nil {
```

Comment:
validator sounded okay, what's the reasoning behind changing it to 'validation'? Everything else is named like the 'doer' of the action e.g. retrier, templater, logger etc.

Reply:
`validator` is a noun. It implies there's some sort of state here. The majority of packages in the standard library don't use `er` names. They aren't 'wrong' but given this package describes a collection of related validation capabilities its peculiar to call it 'validator'.

Reply:
I should point out this isn't absolutely necessary. There's no hard rule here - `validation` seems more appropriate. We can decline the PR 😉